src/main/java/com/example/enterpriseapplication/service/IRecipeService.java

```diff
@@ -2,11 +2,13 @@
 
 import com.example.enterpriseapplication.dto.Recipe;
 
-public interface iRecipeService {
+public interface IRecipeService {
     /**
      * Fetch a recipe with a given ID.
      * @param recipeId a unique identifier for a recipe
      * @return the matching recipe, or null if no matches found
      */
     Recipe fetchById(int recipeId);
+
+    Recipe save(Recipe recipe) throws Exception;
```

Comment:
Implemented save functionality to be used in the service stub as the group intended. They implemented a service method to save which didn't exist at the time.

https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
https://martinfowler.com/bliki/TechnicalDebt.html